pacman-nodejs-app/docs/pacman-nodejs-app-aws-migration.md

```diff
@@ -0,0 +1,532 @@
+# Pac-Man AWS Migration
+
+This tutorial walks you through the steps to perform a Pac-Man migration from an AWS cluster A to a GKE cluster B using GKE for hosting. The tutorial documents the steps
+required to set up cluster A, and documents how to migrate it to cluster B manually or using the automated `kmt` proof-of-concept tool.
+
+## Prerequisites
+
+#### Clone the repository
+
+Follow these steps to [clone the repository](../README.md#clone-this-repository).
+
+#### Create the Pac-Man Container Image
+
+Follow these steps to [create the Pac-Man application container image](../README.md#create-application-container-image).
+
+#### Set up Google Cloud SDK and Push Container Image
+
+Follow these steps to [push the Pac-Man container image to your Google Cloud Container Registry](../README.md#kubernetes-components).
+
+## Setup Your AWS cluster
+Follow [these steps](https://github.com/kubernetes/kops#installing) to install kops
+
+And [these steps](https://github.com/font/k8s-example-apps/blob/master/pacman-nodejs-app/docs/kubernetes-cluster-aws.md#setup-your-aws-environment) to setup your AWS environment.(Stop at the Configure DNS section).
+
+
+## Configure DNS
+
+The kops tool requires a place to build the necessary DNS records in order to build a Kubernetes cluster. There are several scenarios available that you can choose from but in this instance, because we are using google cloud platform to manage our DNS and are migrating to a GKE cluster, we will use scenario 3 as outlined [here](https://github.com/kubernetes/kops/blob/master/docs/aws.md#scenario-3-subdomain-for-clusters-in-route53-leaving-the-domain-at-another-registrar). Create the subdomain with a different name than the one you would ultimately like to use as the public facing address for your application (ex. aws.example.com)
+ 
+Add the NS records to your [Google Cloud Platform](https://cloud.google.com/dns/update-name-servers)
+
+**Make sure to [test your DNS setup](https://github.com/kubernetes/kops/blob/master/docs/aws.md#testing-your-dns-setup) before moving on.**
+
+## Cluster State Storage
+
+We need to create a dedicated S3 bucket for `kops` to manage the state of your cluster. [See here for details](https://github.com/kubernetes/kops/blob/master/docs/aws.md#cluster-state-storage).
+
+## Create the Kubernetes Cluster
```

Comment:
I don't see instructions to create the GKE cluster. We should add instructions to set up both clusters and then migrate from cluster A to cluster B.